decidim-accountability/spec/types/integration_schema_spec.rb

```diff
@@ -36,6 +36,11 @@
           progress
           reference
           startDate
+          address
```

Comment:
please follow alphabetical order 😄

Reply:
done in [99efe17](https://github.com/decidim/decidim/pull/14326/commits/99efe17cd2d5c038ef66c49105c886806373bad0)